third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2830,11 +2830,37 @@ def skew(self, *, numeric_only: bool = False):
         raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
 
     def kurt(self, *, numeric_only: bool = False):
-        """Return unbiased kurtosis over requested axis.
+        """Return unbiased kurtosis over columns.
 
         Kurtosis obtained using Fisher's definition of
         kurtosis (kurtosis of normal == 0.0). Normalized by N-1.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame({"A": [1, 2, 3, 4, 5],
+            ...                     "B": [3, 4, 3, 2, 1],
+            ...                     "C": [2, 2, 3, 2, 2]})
+            >>> df
+                A	B	C
+            0	1	3	2
+            1	2	4	2
+            2	3	3	3
+            3	4	2	2
+            4	5	1	2
+            <BLANKLINE>
+            [5 rows x 3 columns]
+
+        Calculating the kurtosis value of each column.
```

Comment:
[nit] probably more appropriate to end the line with ":" instead of ".". Same elsewhere.

Reply:
[nit] Do we also want to add the examples for `numeric_only = True`? Same for elsewhere.

Reply:
For numeric_only=True, in several Pandas APIs I reviewed, it seems that Pandas doesn't really provide an example for this. In the case of BigFrames, numeric_only=True almost does nothing, except it avoids an error that tells the user to set numeric_only=True. So I'm not quite sure if it's necessary.